docs/rest-api.md

```diff
@@ -0,0 +1,77 @@
+## REST API 
+
+For each RPC service, we define an equivalent HTTP REST API method using [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway). The mapping between service and REST API method can be found in this [file](https://github.com/kubeflow/katib/blob/master/pkg/api/api.proto). The mapping includes the URL path, query parameters and request body. You can read more details on the supported methods and binding options at this [link](https://cloud.google.com/service-infrastructure/docs/service-management/reference/rpc/google.api#http)
+
+## Examples
+Assuming `{HOST}` as the ingress host for vizier-core-rest, 
```

Comment:
Please add a way of using portforward.

kubectl v1.10~
`$ kubectl -n katib port-forward svc/vizier-core-rest 8080:80`

kubectl ~v1.9
`& kubectl -n katib port-forward $(kubectl -n katib get pod -o=name | grep vizier-core-rest | sed -e "s@pods\/@@") 8080:80 `

Reply:
I will add this.